src/main/java/frc/robot/RobotContainer.java

```diff
@@ -153,7 +157,9 @@ private void configureButtonBindings() {
             .whenInactive(new InstantCommand(() -> intakeSubsystem.spin(0), intakeSubsystem), true);
         btnIntakeIn.whenHeld(new InstantCommand(() -> intakeSubsystem.spin(-0.5), intakeSubsystem))
             .whenInactive(new InstantCommand(() -> intakeSubsystem.spin(0), intakeSubsystem), true);
-    }
+
+        btnLED.whenPressed(new InstantCommand(() -> ledDriver.set(ledDriver.autonomous)));
```

Comment:
Where you have `ledDriver.autonomous` on this line, you can change it to `LEDDriver.AUTONOMOUS` to use the Class as a reference point and the capitalized constant name.